src/placeos-rest-api/controllers/users.cr

```diff
@@ -49,6 +49,10 @@ module PlaceOS::Api
     getter user : Model::User do
       lookup = params["id"]
 
+      # TODO: Remove user id query prefixing.
+      # Remove after June 2023, added to help with 2022 user id migration
+      lookup = "#{Model::User.table_name}-#{lookup}" unless lookup.starts_with?("#{Model::User.table_name}-")
+
```

Comment:
```suggestion
```

Reply:
This needs to be moved to line 68 where the query using the user id form takes place